lively.collab/comments/commentMorph.js

```diff
@@ -262,13 +292,17 @@ export class CommentMorph extends Morph {
   performClickAction (action) {
     switch (action) {
       case 'remove':
-        this.referenceMorph.removeComment(this.comment);
+        if (!this.isInEditMode) {
```

Comment:
At this point, it might be more reasonable to implement these as real buttons instead of images, since buttons support all this already

Reply:
Do you mean like e.g. in the browser? In principal I would agree with you. However, I would argue that since the topbar also uses this approach it is a valid strategy in lively and the use of buttons instead of our labels would be a different refactoring ticket? (Just to be clear: I am in favor of doing this, but I think we could merge this anyway).